### DISCUSSION TITLE
How to change selection color?

### DISCUSSION THREAD

**Discussion by Gnurrf** [RyotaUshio/obsidian-pdf-plus#117] [Q&A] [answered]
In the sandbox vault, I have a selection color in PDF++ well visible on a white PDF page background. In my own vault, based on the ITS theme, the selection color is barely visible (a _very_ light grey or blue?). Is there a CSS snippet which would allow me to change the selection color for better visibility?

**Comment by RyotaUshio** [comment, marked as answer]
You can target `.textLayer ::selection`, but you don't need to write CSS snippets.
Open Style Settings and go to ITS Theme Settings > Colors > Highlights > Text Selection.

**Reply by Gnurrf** [reply]
Many thanks for the reply, that is indeed much easier then adding CSS snippets :-)
